develop-docs/sdk/telemetry/traces/index.mdx

```diff
@@ -228,16 +228,22 @@ A transaction's sampling decision should be passed to all of its children, inclu
 
-The random value (`sample_rand`) is set according to the following rules:
-
-1. When tracing is enabled and a new trace is started, the SDK generates a `sample_rand` value for the current execution context. A `sample_rand` is a float (`0.1234` notation) in the range of `[0, 1)` (including 0.0, excluding 1.0).
-  1. This also applies to Tracing without Performance where we simply want to generate `sample_rand` every time a new trace is started. This `sample_rand` value can then be stored on `PropagationContext`.
-1. It is _recommended_ to generate the random number deterministically using the trace ID as seed or source of randomness (make sure to do this atomically so multiple threads accessing the same random instance don't mix their seeds). The exact method by which the random number is created is implementation-defined and may vary between SDK implementations.
-1. The `sample_rand` is part of the DSC (Dynamic Sampling Context) and as with other values on the `baggage` header, the `sample_rand` value from the current execution context should be propagated to downstream SDKs. It should also be sent to other system as part of the `baggage` header if Performance is disabled and sampling decision is deferred.
-1. On incoming traces, an SDK takes the incoming `sentry-sample_rand` value in the `baggage` header and uses it for the rest of the current execution context (for example, request) by storing it in the `PropagationContext`.
-1. If `sample_rand` is missing on an incoming trace, the SDK creates a new one applying **special rules** and uses it for the current execution context:
-   1. If `sample_rate` (inside `baggage`) and the sampling decision (trailing `-1` or `-0` from the `sentry-trace` header) are present in the incoming trace, create `sample_rand` so that when compared to the incoming `sample_rate` it would lead to the same sampling decision that is in the `sentry-trace` header. This means, for a decision of `True` generate a random number in half-open range `[0, rate)` and for a decision of `False` generate a random number in range `[rate, 1)`.
-   1. If the sampling decision is missing on the incoming trace, generate a random number in range of `[0, 1)` (including 0.0, excluding 1.0), like for a new trace.
+SDKs must set and propagate the `sample_rand` as follows:
+
+1. Whenever a new trace is started, including in the Tracing without Performance case, the SDK generates a `sample_rand` value for the **entire trace**.
+   1. SDKs must generate the `sample_rand` by pseudo-randomly sampling a floating-point number from the uniform distribution over the range `[0.0, 1.0)` (including `0.0`, excluding `1.0`).
```

Comment:
I think it would be easier to say that we're generating a pseudorandom number, rather than sampling one, to not confuse this with the actual sampling that occurs afterwards. However, is it even necessary to call out "pseudo-random" here if in the last sub-bullet it recommends a deterministic number? An implementation doesn't have to follow that recommendation and we actually don't care if the number is pseudorandom or true random.

Same for the "uniform distribution". This is correct and I think this is good to add for correctness, but I wonder if this just adds complexity to the spec given that a uniform distribution is implied when one says "generate a random number".

Reply:
+1 - it read overly and a bit unnecessarily clever. "generate a random number in the range ..." feels good enough